tools/release-info/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@hypermode/release-info",
+  "version": "",
+  "description": "Generates release information for the Modus CLI",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "type": "module",
+  "scripts": {
+    "build": "rm -rf dist && tsc -b",
+    "watch": "rm -rf dist && tsc -b -w"
+  },
+  "repository": "github:hypermodeinc/modus",
+  "author": "Hypermode Inc.",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/hypermodeinc/modus/issues"
+  },
+  "homepage": "https://github.com/hypermodeinc/modus",
+  "dependencies": {
+    "semver": "^7.6.3"
+  },
+  "devDependencies": {
```

Comment:
Let's add eslint and corresponding lint and build workflows for this tool.  Thanks